### ISSUE TITLE
Feature request / idea - alias / nickname / custom icon for nitrokey

### ISSUE THREAD

**Issue by kizm0** [Nitrokey/nitrokey-app2#285]
Hello,

i do use several nitrokeys with the NK-App2, with the time i remembered the order of my nitrokeys witch are listed in the app, but i thought it could be helpful if you can define an alias or nickname for a nitrokey in the settings tab which will be displayed on the left pane (the left pane should be resizable) or a custom image/icon. The nickname / icon (base64) could be stored directly on the nitrokey or on the device (app) linked with the uuid...

**Comment by daringer** [comment]
yes, this kind of fell of the cliff once we introduced the slim bar on the left instead of having the full bar with the nitrokey uuid ... I'd suggest to first find space inside the icon to put some characters there and then on top allow something like a "label" to be set for a nitrokey - but this spawns the entire chain of saving a state (config) which isn't done yet ... so first things first, but this here will be 2nd

**Comment by qwertologe** [comment]
It is very unfortunate that the models look the same in the GUI, have the same model name, and only differ in `Variant`. However, the two keys here (`Nitrokey 3A Mini` and `Nitrokey 3C NFC`) look different and have different names.

At the very least, the standard icon and model name at the top of `Overview` should match the used Nitrokey variant.

I also think the label idea is great!

**Comment by ABuljko** [comment]
We could  also add a feature to the app2 that would allow the user to change the color block for a specific nitrokey since it would help the user differentiate between multiple keys (we can keep it in the settings tab) 

<img width="107" height="137" alt="Image" src="https://github.com/user-attachments/assets/1b93b8c2-e349-486a-a112-6ca9729c3b1b" />
